release/models/wifi/openconfig-wifi-types.yang

```diff
@@ -248,6 +254,27 @@ module openconfig-wifi-types {
       each frequency.";
   }
 
+  identity FREQ_6GHZ {
+    base OPERATING_FREQUENCY;
+    description "The Radio or SSID will operate at 6GHz only.";
+  }
+
+  identity FREQ_5_6GHZ {
+    base OPERATING_FREQUENCY;
+    description "The Radio or SSID will be dual-band; operating in both 5 &
+    6GHz frequencies. Dual-band Radio typically refers to a Monitor-mode
+    radio, hopping between frequencies, dwelling for a configurable amount of
+    time on each frequency.";
+  }
+
+  identity FREQ_2_5_6GHZ {
+    base OPERATING_FREQUENCY;
+    description "The Radio or SSID will be tri-band; operating in 2.4, 5 and
+    6GHz frequencies. Tri-band Radio typically refers to a Monitor-mode radio,
```

Comment:
As a note, I don't recall why we have the "typically refers to mon-mode radio..." bits in the description. I think it's often enough to have a dual-band SSID (and now perhaps a tri-band SSID) that saying "Typically this refers to mon-mode..." might no longer make sense.